tests/test_main_shutdown_coverage.py

```
@@ -35,6 +35,25 @@ async def _async_return(*_args, **_kwargs):
     return _async_return
 
 
+class _ImmediateAwaitable:
+    """Lightweight awaitable that resolves immediately without creating coroutines."""
+
+    def __init__(self, value=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)